app/actions/contactsActions.js

```diff
@@ -5,13 +5,79 @@ import { has, isEmpty, keys, values, indexOf, zipObject, omit } from 'lodash-es'
 
 import { getStorage, setStorage } from '../core/storage'
 
+export const ID = 'contacts'
+
+export type TChains = { contactKey: string, chain: string }[]
```

Comment:
Dont we need name and address in here too 🤔

Reply:
don't is necessary, the addressBook storage just is a "auxiliary table"